backend/app/models/spree/backend_configuration.rb

```diff
@@ -1,9 +1,24 @@
 # frozen_string_literal: true
 
+require 'spree/backend/environment/product_tabs'
+
 module Spree
   class BackendConfiguration < Preferences::Configuration
     preference :locale, :string, default: Rails.application.config.i18n.default_locale
 
+    def product_tabs
+      @product_tabs ||= Backend::Environment::ProductTabs.new.tap do |product_tabs|
```

Comment:
I can probably do something like this instead: https://github.com/solidusio/solidus/pull/3232/files#diff-c379205c70b6f279fcb4141310bbef4dR9

cc @elia